src/openrct2/paint/tile_element/Surface.cpp

```diff
@@ -604,29 +604,29 @@ static void viewport_surface_draw_tile_side_bottom(paint_session * session, enum
     }
 
     if (isWater)
-        regs.dl = height;
+        dl = height;
 
     if (neighbour.tile_element == nullptr)
     {
-        regs.ch = 1;
-        regs.ah = 1;
+        ch = 1;
+        ah = 1;
     }
     else
     {
         if (isWater)
         {
-            regs.dh = map_get_water_height(neighbour.tile_element);
-            if (regs.dl == regs.dh)
+            dh = map_get_water_height(neighbour.tile_element);
+            if (dl == dh)
```

Comment:
AppVeyor chokes on the possibility uninitialised usage of `dl`. Should perhaps default to 0.

Reply:
done